app/views/components/input_text.scala.html

```diff
@@ -0,0 +1,46 @@
+@*
```

Comment:
I don't think we need to create a custom helper for inputting text, there should be a default one present.